### PR TITLE
feat: improve resiliency to w3f archives

### PR DESCRIPTION
**Pull request by mkykadir**

### CODE REVIEW

File: src/lib/uploader/Web3FunctionUploader.ts [hunk 1]
```diff
@@ -9,6 +9,9 @@ import { Web3FunctionSchema } from "../types";
 
 const OPS_USER_API =
   process.env.OPS_USER_API ?? "https://api.gelato.digital/automate/users";
+
+const MAX_SIZE = 10 * 1024 * 1024; // 10 MB
```
Comment (goums) [inline src/lib/uploader/Web3FunctionUploader.ts:13]: let's use 2 different limits:
- DOWNLOAD_MAX_SIZE: 1mb
- EXTRACT_MAX_SIZE: 10mb 

as the compressed tarball should be limited to 1mb

**Review by goums — CHANGES_REQUESTED** [review]

File: src/lib/uploader/Web3FunctionUploader.ts (hunk 1 above)
Comment (mkykadir) [reply src/lib/uploader/Web3FunctionUploader.ts:13]: @goums on the tests that I was fetching cancelled/active tasks I was remembering 4 MB archives, why we're limiting to 1 MB?

Comment (mkykadir) [reply src/lib/uploader/Web3FunctionUploader.ts]: @goums separated

Comment (goums) [reply src/lib/uploader/Web3FunctionUploader.ts]: @mkykadir we are limiting archives to 1mb on user api to have DoS vector on on our DB storage

**Review by goums — APPROVED** [review]

**Pull request merged**